### DISCUSSION TITLE
Running in Docker / Building the Dockerfile

### DISCUSSION THREAD

**Discussion by harrisonj94** [microsoft/azuredatastudio#24730] [Q&A]
I would like to build the image here build/azure-pipelines/linux/Dockerfile for use in Kasm, as a workspace, I would like to know why this is not published as a package on the repo, and is it possible to prepopulate this image with connections to multiple sql servers so that on use it will have them preloaded.

**Comment by erinstellato-ms** [comment]
@harrisonj94 To confirm I understand your question - you're asking to have Azure Data studio as part of the image, and then have the install of Azure Data Studio pre-populated with a set of connections?
